cdap-docs/developers-manual/source/testing/debugging.rst

```diff
@@ -110,6 +110,14 @@ The corresponding HTTP requests for the ``RetrieveCounts`` service of this appli
 
 Analysis of the response would give you the host names and debugging ports for all instances of the service.
 
+If you are trying to debug a system service, this endpoint gives the container info for a system service::
```

Comment:
I don't think this belongs here. We wouldn't expect people to debug a system service, as that is CDAP code and not application code.